packages/cli/test/integration/controllers/dynamic-node-parameters.controller.test.ts

```diff
@@ -0,0 +1,80 @@
+import type { SuperTest, Test } from 'supertest';
+import { createOwner } from '../shared/db/users';
+import { setupTestServer } from '../shared/utils';
+import * as AdditionalData from '@/WorkflowExecuteAdditionalData';
+import type {
+	INodeListSearchResult,
+	IWorkflowExecuteAdditionalData,
+	ResourceMapperFields,
+} from 'n8n-workflow';
+import { mock } from 'jest-mock-extended';
+import { DynamicNodeParametersService } from '@/services/dynamicNodeParameters.service';
+
+describe('DynamicNodeParametersController', () => {
```

Comment:
I was thinking about adding unit tests instead, since there are plenty of logical checks in the controller itself.
but, I can help with that bit.

Reply:
Thanks, this fix is about switching to `POST` and `body` hence I thought of integration tests.